wire-ios/Wire-iOS/Resources/Configuration/Version.xcconfig

```diff
@@ -17,3 +17,4 @@
 //
 
 WIRE_SHORT_VERSION = 3.122.0
+MAJOR_VERSION = 0
```

Comment:
This fixes an error with Xcode 16.3.
Since we define 
```
DYLIB_COMPATIBILITY_VERSION = $(MAJOR_VERSION).0
```
but WireCommonComponents didn't have `MAJOR_VERSION` set, the resulting value for `DYLIB_COMPATIBILITY_VERSION` was ".0" which is not a correct version string (compile error).

Reply:
@netbe please let me know if you have objections to this change.

Reply:
so it's 0.0?

Reply:
Exactly.